components/dropdown_input.tsx

```diff
@@ -9,7 +9,7 @@ import './dropdown_input.scss';
 
 // TODO: This component needs work, should not be used outside of AddressInfo until this comment is removed.
```

Comment:
👀

Reply:
Didn't even see that 😂 . Asked Devin (who wrote the comment) if he has any concerns